src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -174,6 +174,48 @@ public static void ClientBaseOfT_Sync_RoundTrip_Call_Using_NetTcpTransport()
         }
     }
 
+    [Fact]
+    [OuterLoop]
+    public static void OperationContextScope_HttpRequestCustomMessageHeader_RoundTrip_Verify()
+    {
+        BasicHttpBinding binding = new BasicHttpBinding();
+        MyClientBase<IWcfService> client = new MyClientBase<IWcfService>(binding, new EndpointAddress(Endpoints.HttpBaseAddress_Basic));
+        IWcfService serviceProxy = client.ChannelFactory.CreateChannel();
+
+        string customHeaderName = "OperationContextScopeCustomHeader";
+        string customHeaderNS = "http://Microsoft.WCF.OperationContextScope_HttpRequestCustomMessageHeader_RoundTrip_Verify";
+        string customHeaderValue = "CustomHappyValue";
```

Comment:
Does the namespace need to be exactly this string?  I think we should probably adopt a convention of using tempuri for this kind of usage to show it's not a real namespace or URI.